src/shared/utils/duration.ts

```diff
@@ -136,16 +136,18 @@ export const convertTimeRangeToCustom = (
   let lower = ''
 
   if (timeRange.type === 'selectable-duration') {
-    lower = moment()
-      .subtract(timeRange.seconds, 's')
-      .toISOString()
+    const lowerDate = new Date()
+    lowerDate.setSeconds(new Date().getSeconds() - timeRange.seconds)
```

Comment:
when you `setSeconds` or `setMilliseconds` it's fine to use `lowerDate.getSeconds()` or `lowerDate.getMilliseconds()` instead of getting them from a `new Date()`

Reply:
Got it! i'll make that change!